catppuccin-steam.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash 
+set -euo pipefail
+IFS=$'\n\t'
+
+# Install dependencies
+if [[ -f /usr/bin/pacman  && ! -f {/usr/bin/wget,/usr/bin/unzip,/usr/bin/git} ]]
+then
+  sudo pacman -S wget unzip git
+elif [[ -f /usr/bin/emerge && ! -f {/usr/bin/wget,/usr/bin/unzip,/usr/bin/git} ]]
+then
+  sudo emerge -av wget unzip git
+elif [[ -f /usr/bin/apt && ! {/usr/bin/wget,/usr/bin/unzip,/usr/bin/git} ]]
+then
+   sudo apt-get install wget unzip git
+elif [[ -d /nix ]]
+then
+  nix-shell -p wget unzip git
+elif [[ -f /usr/bin/dnf && ! -f {/usr/bin/wget,/usr/bin/unzip,/usr/bin/git} ]]
+then
+  sudo dnf install wget unzip git
+else
+  readf "Please install the following packages using your package manager:\n curl\n wget\n unzip\n git\n"
+  exit
+fi
+
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Get the latest release of SFP and unzip it
+mkdir SFP 
+cd SFP || exit 1
+wget https://github.com/PhantomGamers/SFP/releases/download/0.0.14/SFP_UI-SelfContained-linux-x64.zip
+unzip *.zip
```

Comment:
```suggestion
unzip ./*zip
```
SC2035: Use `./*glob*` or `-- *glob*` so names with dashes won't become options.